packages/pangea-node-sdk/tests/integration/vault.test.ts

```diff
@@ -484,16 +480,24 @@ it("RSA encrypting life cycle", async () => {
 });
 
 it("AES encrypting life cycle", async () => {
-  const algorithm = Vault.SymmetricAlgorithm.AES;
+  const algorithms = [
+    Vault.SymmetricAlgorithm.AES128_CBC,
+    Vault.SymmetricAlgorithm.AES256_CBC,
+    Vault.SymmetricAlgorithm.AES128_CFB,
+    Vault.SymmetricAlgorithm.AES256_CFB,
+    Vault.SymmetricAlgorithm.AES256_GCM,
+  ];
   const purpose = Vault.KeyPurpose.ENCRYPTION;
-  try {
-    const id = await symGenerateDefault(algorithm, purpose);
-    await encryptingCycle(id);
-    await vault.delete(id);
-  } catch (e) {
-    console.log(`Failed symmetric encrypting life cycle with ${algorithm} and ${purpose}`);
-    expect(false).toBeTruthy();
-  }
+  algorithms.forEach(async (algorithm) => {
+    try {
+      const id = await symGenerateDefault(algorithm, purpose);
+      await encryptingCycle(id);
+      await vault.delete(id);
+    } catch (e) {
+      console.log(`Failed symmetric encrypting life cycle with ${algorithm} and ${purpose}`);
+      expect(false).toBeTruthy();
```

Comment:
I know this is done elsewhere, and you don't have to change all that, but I think it would be better to just rethrow the error and let Jest display it, because right now it would be unnecessarily difficult to debug an error here.
```suggestion
      throw e;
```

Reply:
Updated in this tests. I'll be updating the other ones as we work on them